src/js/components/Tree/Tree.js

```diff
@@ -0,0 +1,161 @@
+import React, { useContext, useState, useEffect } from 'react';
+import { ThemeContext } from 'styled-components';
+import { FormDown } from 'grommet-icons/icons/FormDown';
+import { FormNext } from 'grommet-icons/icons/FormNext';
+import { FormPrevious } from 'grommet-icons/icons/FormPrevious';
+
+import { Box } from '../Box';
+import { Button } from '../Button';
+import { List } from '../List';
+import { Text } from '../Text';
+
+const Tree = ({ children, mode, data }) => {
+  const theme = useContext(ThemeContext);
+
+  const [depth, setDepth] = useState(0);
+  // an array of the path names
+  const [breadcrumbs, setBreadcrumbs] = useState([]);
+  const [selectedNode, setSelectedNode] = useState();
+  // treeLevel should only be updated on useEffect to reflect correct depth
+  const [treeLevel, setTreeLevel] = useState(data);
```

Comment:
Initializing the "level" of the tree directly from the "data" looks odd to me here, conceptually.

Reply:
I agree, it more about the data of the level rather than the level itself.
would `treeLevelData` make more sense?